sfaira/versions/metadata/base.py

```diff
@@ -347,17 +347,24 @@ def get_descendants(self, node: str) -> List[str]:
         node = self.convert_to_id(node)
         return list(networkx.descendants(self.graph, node))
 
-    def is_a(self, query: str, reference: str) -> bool:
+    def is_a(self, query: str, reference: str, convert_to_id=True) -> bool:
```

Comment:
add into typing that query can be list `query: Union[str, List[str]]`

Reply:
This is not necessary anymore in the updated version